lib/tasks/lead_provider_token.rake

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+namespace :lead_provider do
+  desc "creates API token for lead provider ID parameter"
+  task generate_token: :environment do
+    begin
+      lead_provider = LeadProvider.find(ARGV[1])
+    rescue StandardError
+      lead_provider = LeadProvider.find_by(name: ARGV[1])
+    end
+    token = LeadProviderApiToken.create_with_random_token!(lead_provider: lead_provider)
+    puts "Generated API token for lead provider (#{lead_provider.id}): #{token}"
```

Comment:
Maybe use Rails logger for it, and... Will that get logged in our logs? Are we sure this is the safest way of retrieving the token?

Reply:
We don't know how this is going to work for production yet so this is just a starting point that we can iterate on.